tests/src/ut_test_kvp.c

```diff
@@ -199,37 +226,49 @@ void test_ut_kvp_open_malloced( void )
     ut_kvp_close(pInstance);
 
     /* Negative Read Test, NULL PARAM */
-    UT_LOG_STEP("ut_kvp_open( pInstance, NULL ) - Negative");
-    status = ut_kvp_open( pInstance, NULL, true );
+    UT_LOG_STEP("ut_kvp_openMemory( pInstance, NULL ) - Negative");
+    status = ut_kvp_openMemory( pInstance, NULL, -1);
     UT_ASSERT( status == UT_KVP_STATUS_INVALID_PARAM );
 
     /* data doesn't exist */
-    UT_LOG_STEP("ut_kvp_open( pInstance, %s - filename doesn't exist ) - Negative", KVP_VALID_TEST_NO_FILE);
+    UT_LOG_STEP("ut_kvp_openMemory( pInstance, %s - filename doesn't exist ) - Negative", KVP_VALID_TEST_NO_FILE);
```

Comment:
you can't have a filename doent exist when the function doesn't take a filename?